install.sh

```diff
@@ -104,17 +69,37 @@ else
   echo_info "Unable to detect system information, installation will continue."
 fi
 
+# Check if the user is on ChromeOS v117+ and not in the VT-2 console, sudo will not work
+if sudo | grep -q 'no new privileges'; then
```

Comment:
Calling `sudo` with no arguments will print the help message, this might works:
```shell
if [[ "$(sudo /bin/true 2>&1)" == *'no new privileges'* ]]
```

Reply:
When the relevant flag is set, it will print the warning instead (at least in my testing in docker containers)